plugin.xml

```diff
@@ -21,6 +21,7 @@
 
   <engines>
     <engine name="cordova" version=">=3.0.0"/>
+    <engine name="cordova-ios" version=">=4.3.0" />
```

Comment:
I think that is a good idea as long as those `config-file` lines get ignored gracefully. I'll try and test it out `cordova-ios` < 4.3.0